recipes/clincnv/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "clincnv" %}
+{% set version = "1.18.3" %}
+
+package:
+  name: {{name}}
+  version: {{ version }}
+
+
+source:
+  url: https://github.com/imgag/ClinCNV/archive/refs/tags/{{ version }}.zip
+  sha256: "b637c61e7541ca3ae11198c681d8b8e56e9fba276b0c7c7b6c08c363dc568987"
+
+build:
+  noarch: generic
+  number: 0
+  run_exports:
+        - {{ pin_subpackage('clincnv', max_pin="x.x") }}
+
+requirements:
+  host:
+    - r-base >=4.0
+    - r-optparse
+    - r-robustbase
+    - r-mass
+    - r-data.table
+    - r-foreach
+    - r-doparallel
+    - r-mclust
+    - r-r.utils
+    - r-rcolorbrewer
+    - r-party
+    - r-umap
+    - r-dbscan
+    - r-rcpp
+  run:
+    - r-base >=4.0
+    - r-optparse
+    - r-robustbase
+    - r-mass
+    - r-data.table
+    - r-foreach
+    - r-doparallel
+    - r-mclust
+    - r-r.utils
+    - r-rcolorbrewer
+    - r-party
+    - r-umap
+    - r-dbscan
+    - r-rcpp
+
+test:
+  commands:
+    - clinCNV.R --help
+    - mergeFilesFromFolder.R --help
+    - mergeFilesFromFolderDT.R --help
+
+about:
+  home: https://github.com/imgag/ClinCNV
+  license: MIT
+  license_file: LICENSE.txt
```

Use tarball license:

```suggestion
  license_file: LICENSE
```